docs/user/advanced.rst

```diff
@@ -940,7 +937,7 @@ If the remote server is very slow, you can tell Requests to wait forever for
 a response, by passing None as a timeout value and then retrieving a cup of
 coffee.
 
-.. code-block:: python
+::
```

Comment:
:: produces the same output and is used everywhere else in the documentation. I couldn't find any additional HTML markup generated by `.. code-block:: python` v.s. `::` and the [documentation](http://www.sphinx-doc.org/en/stable/markup/code.html) states it's only necessary when displaying a language not in the default config (which is python for Requests). 

It looks like this all came from the same commit from Kevin though, so he may have a reason for this that I'm missing.